actors/util/adt/store.go

```diff
@@ -5,17 +5,19 @@ import (
 
 	"github.com/filecoin-project/go-state-types/cbor"
 	"github.com/filecoin-project/go-state-types/exitcode"
+	adt2 "github.com/filecoin-project/specs-actors/v2/actors/util/adt"
 	cid "github.com/ipfs/go-cid"
 	ipldcbor "github.com/ipfs/go-ipld-cbor"
 
 	vmr "github.com/filecoin-project/specs-actors/v3/actors/runtime"
 )
 
 // Store defines an interface required to back the ADTs in this package.
-type Store interface {
-	Context() context.Context
-	ipldcbor.IpldStore
-}
+// type Store interface {
+// 	Context() context.Context
+// 	ipldcbor.IpldStore
+// }
```

Comment:
👍 . You can delete the commented code.

Maybe even go back to alias the v2 store onto v0.